crates/cxx-qt-build/src/lib.rs

```diff
@@ -1019,17 +1005,18 @@ impl CxxQtBuilder {
     }
 
     fn write_interface_include_dirs(&self) {
-        if let Some(interface) = &self.public_interface {
-            for (header_dir, symlink) in &interface.exported_include_directories {
-                Self::symlink_directory(header_dir, dir::header_root().join(symlink))
-                    .unwrap_or_else(|_| {
-                        panic!(
-                            "Failed to create symlink `{}` for export_include_directory: {}",
-                            symlink,
-                            header_dir.to_string_lossy()
-                        )
-                    });
-            }
+        let Some(interface) = &self.public_interface else {
+            return;
+        };
```

Comment:
Huh, I had no idea this was a thing :sweat_smile: 

Apparently this has been stable since Rust 1.65: https://doc.rust-lang.org/rust-by-example/flow_control/let_else.html
The rust-version in our main Cargo.toml says we depend on Rust 1.77.2, so we can use this feature, nice!